prelude/slang-cpp-types.h

```diff
@@ -41,6 +41,26 @@ struct Array
     size_t count;
 };
 
+template<size_t N>
+struct AnyValue
+{
+    uint8_t data[N];
```

Comment:
I was going to ask about alignment constraints (what if the user wants to store a `double` in here?) but I then realized that we don't care about alignment for these buffers at all: they are only accessed by copying in/out, rather than by casting the memory to the desired type.

Some of this policy stuff really needs to be handled by adding comments to the declaration of this type and the pack/unpack operations.

Reply:
This is supposed to be temporary code. All these `AnyValue` related stuff in prelude will go away when I have the packing pass in place.